### ISSUE TITLE
Terminal font installation

### ISSUE THREAD

**Issue by PaulGrandperrin** [t184256/nix-on-droid#180]
Because I cannot reopen issue #120 , I open a new one to continue the discussion.

[nix-darwin](https://github.com/LnL7/nix-darwin/blob/master/modules/fonts/default.nix?rgh-link-date=2022-04-30T21%3A50%3A39Z#L55) and [home-manager](https://github.com/nix-community/home-manager/blob/781d25b315def05cd7ede3765226c54216f0b1fe/modules/targets/darwin/fonts.nix#L14=) also need to copy fonts over filesystems boundaries and use the activation script to do it.

Let's do the same?

I can write a PR if I get the green light.

In the meantime, here what it looks like (when using `home-manager`): 
```nix
home.activation = {
  copyFont = let
      font_src = "${pkgs.nerdfonts.override { fonts = [ "FiraCode" ]; }}/share/fonts/truetype/NerdFonts/Fira Code Regular Nerd Font Complete Mono.ttf";
      font_dst = "${config.home.homeDirectory}/.termux/font.ttf";
    in lib.hm.dag.entryAfter ["writeBoundary"] ''
      ( test ! -e "${font_dst}" || test $(sha1sum "${font_src}"|cut -d' ' -f1 ) != $(sha1sum "${font_dst}" |cut -d' ' -f1)) && $DRY_RUN_CMD install $VERBOSE_ARG -D "${font_src}" "${font_dst}"
  '';
};
```

Other examples are available at #120

**Comment by t184256** [comment]
I'm not against in principle, but probably will be of little help since my development machine suddenly decided to go to silicon heaven.

What's the characteristic execution time for such a snippet?

**Comment by Gerschtli** [comment]
A simple option to specify a font would be a great addition. Would be worth creating a MR for that :)

**Comment by Gerschtli** [comment]
This issue seems to be solved by #201. Feel free to reopen if you think otherwise :)
